axlearn/common/attention.py

```diff
@@ -3891,6 +3904,7 @@ def build_remat_spec(
         stack_cfg: A transformer config.
         self_attention: Checkpoint self attention layer activations if true.
         feed_forward: Checkpoint feed-forward layer activations if true.
+        offload_dst: Destination of remat checkptoing offloading.
```

Comment:
Add a link to the JAX documentation on `offset_dst` on the potential values?

Reply:
Sounds good. Thanks!

Reply:
Was this change pushed?

Reply:
No ah I misunderstood Ruoming's comment and thought he was fine there being no docs. Let me add the link to maxtext as a comment.

Reply:
Done